### PR TITLE
Implement `sendCommand` in `WebSocketCockpitConnector`

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/gravitee-cockpit/issues/323

**Description**

### CODE REVIEW

File: gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    private Single<Account> createAccount(User user) {
+        Faker faker = new Faker();
+        NewAccount newAccount = new NewAccount(faker.name().name(), UUID.randomUUID().toString(), faker.shakespeare().asYouLikeItQuote());
+
+        return webClient
+            .postAbs(BASE_URL + "/management/accounts")
+            .basicAuthentication(user.getUsername(), user.getPassword())
+            .rxSendJson(newAccount)
+            .map(HttpResponse::bodyAsJsonObject)
+            .map(json -> new Account().withId(json.getString("id")));
```
Comment (loriepisicchio) [inline gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java:186]: Is it possible to parse the json response and get real data in account object ? Don't you think we might need the other fields at some point ?

Comment (gaetanmaisse) [reply gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java]: 👍🏻 I used `Json.Decode` to autoparse the object with a minimal effort.

File: gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+        return environment;
+    }
+
+    @Bean
+    public HttpClientFactory httpClientFactory(Vertx vertx, Environment environment) {
+        HttpClientConfiguration configuration = new HttpClientConfiguration(environment);
+
+        // First we need a user
+        User user = createUser().blockingGet();
```
[the remark this answers is not in the document]
Comment (gaetanmaisse) [reply gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java]: Done

File: gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+import org.springframework.lang.NonNull;
+
+@Configuration
+@ComponentScan(
+    basePackages = "io.gravitee.cockpit.connectors.ws",
+    excludeFilters = { @ComponentScan.Filter(type = FilterType.REGEX, pattern = "io.gravitee.cockpit.connectors.ws.spring") }
+)
+class TestInitializer {
+
+    private static final String BASE_URL = "http://localhost:8063";
```
[the remark this answers is not in the document]
Comment (gaetanmaisse) [reply gravitee-cockpit-connectors-test/src/test/java/io/gravitee/cockpit/connectors/test/TestInitializer.java]: Done, I added a Readme to explain how to run the tests and override the default values for both HTTP and WS endpoints.

**Review by loriepisicchio — APPROVED** [review]

**Review by jgiovaresco — APPROVED** [review]

**Pull request merged**